device/cuda/src/cca/component_connection.cu

```diff
@@ -963,7 +963,7 @@ __global__ void partition_sorting_kernel(unsigned* out,
 }
 
 std::tuple<vecmem::unique_alloc_ptr<unsigned[]>, std::size_t> partition_gpu(
-    const cell_container_types::host& data, vecmem::memory_resource& mem,
+    const cell_container_types::host&, vecmem::memory_resource& mem,
```

Comment:
```suggestion
    vecmem::memory_resource& mem,
```
I don't see why not just remove this entirely

Reply:
This way it shares the same API as the `partition_cpu` function. Not sure if we should keep them both.

Reply:
I see. I like keeping both in, since this is sort of a separate thing to our main example chains. Might as well leave it